src/pages/Content/index.js

```diff
@@ -13,3 +13,11 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
```

Comment:
Oh, I see. Yul already worked on this. @YukiChung if possible, please don't forget to post the branch name on the task on trello. I didn't realize you had already worked on this, and I have created a branch and posted there. Looks like we are working in parallel. Let me delete my branch. By the way, Agam mentioned that you may be using old version of node.js and it may be causing the update to package-lock.json file. I think you can update your gitignore file to exlude this file and it will not push the package-lock.json again.